Tests/Clappr_Tests/MediaOptionTests.swift

```diff
@@ -0,0 +1,49 @@
+import Quick
+import Nimble
+
+@testable import Clappr
+
+class MediaOptionTests: QuickSpec {
+    override func spec() {
+        describe("MediaOption") {
+            describe("#init") {
+                var name: String!
+                var type: MediaOptionType!
+                var language: String!
+                var raw: String!
+                var mediaOption: MediaOption!
+
+                beforeEach {
+                    name = "name"
+                    type = MediaOptionType.audioSource
+                    language = "language"
+                    raw = "raw"
+                    mediaOption = MediaOption(name: name, type: type, language: language, raw: raw as AnyObject)
+                }
+
+                it("has name") {
+                    expect(mediaOption.name) == name
```

Comment:
We've been using `.to(equal(something))`. I recommend to follow this pattern to avoid inconsistencies in the code.